tests/test_s3_to_json_s3.py

```diff
@@ -445,12 +445,8 @@ def test_update_json_schemas(self, s3_obj, archive_map):
         )
         assert isinstance(json_schemas, list)
         file_names = [
-            "taskData",
             "taskData.json",
-            "info.json",
             "motion.json",
-            "weather.json",
-            "taskResult.json",
```

Comment:
These files are in the mocked `s3_obj`, but not in the mocked `archive_map`. Now that we are no longer including files without JSON schemas in `json_schemas`, we don't expect these files.

Reply:
Yes, but not for the `update_json_schemas` function. If a file doesn't have a JSON Schema, we either ignore it or map it using `dataset_mapping.yaml` (if we can find it there). We test that functionality in `test_get_dataset_identifier_does_not_have_schema`.